igvc_ws/src/igvc_nav/src/igvc_nav_node.py

```diff
@@ -52,20 +85,34 @@ def timer_callback(event):
         delta = heading_to_la - heading
         delta = (delta + 180) % 360 - 180
 
-        motor_pkt = motors()
-        motor_pkt.left = 0.4 + 0.6 * (delta / 180)
-        motor_pkt.right = 0.4 - 0.6 * (delta / 180)
+        # print('diff', delta)
 
+        percent_bad = delta/180
+
+        motor_pkt = motors()
+        motor_pkt.left = 1.0 * (1 - abs(percent_bad))**5 - 0.6 * percent_bad
+        motor_pkt.right = 1.0 * (1 - abs(percent_bad))**5 + 0.6 * percent_bad
```

Comment:
Left and right are just the desired linear velocities of the robot's wheels (currently linear velocities instead of angular velocities bcuz that's what the Nucleo motor driver firmware wants and the sim also currently expects that. Need to fix these). 1.0 m/s is just an arbitrary speed that I liked (2.2 mph)